doc/source/whatsnew/v2.1.0.rst

```diff
@@ -183,10 +183,13 @@ Notable bug fixes
 
 These are bug fixes that might have notable behavior changes.
 
-.. _whatsnew_210.notable_bug_fixes.notable_bug_fix1:
+.. _fixed_groupby_error_with_nullable_types:
 
-notable_bug_fix1
+groupby_nullable_types_error
```

Comment:
I don't think this qualifies for a notable bug fix. Instead, could you add to the GroupBy subsection of the bugfixes down below this.